scipy/signal/filter_design.py

```diff
@@ -2063,7 +2067,7 @@ def _zpkbilinear(z, p, k, fs):
     return z_z, p_z, k_z
 
 
-def _zpklp2lp(z, p, k, wo=1.0):
+def lp2lp_zpk(z, p, k, wo=1.0):
```

Comment:
It would be good to have reciprocal `See Also` entries for the `zpk` and `(b, a)` versions

Reply:
Ok, added